core/pkg/monitor/gpu_nvidia.go

```diff
@@ -99,21 +107,21 @@ func NewGPUNvidia(logger *observability.CoreLogger, pid int32, samplingInterval
 	}
 
 	if err := g.cmd.Start(); err != nil {
-		// this is a relevant error, so we will report it to sentry
+		// This is a relevant error, so we will report it to sentry.
```

Comment:
nit: this comment doesn't seem useful; the fact that the line is in the code should imply that it's relevant